drivers/peci/peci_ite_it8xxx2.c

```diff
@@ -0,0 +1,370 @@
+	uint32_t bitrate;
+};
+
+static const struct peci_alts_cfg
+	peci_alt_inst[DT_INST_NUM_PINCTRLS_BY_IDX(0, 0)] =
+	IT8XXX2_DT_ALT_ITEMS_LIST(0);
+
+static const struct peci_it8xxx2_config peci_it8xxx2_config0 = {
+	.base_addr = DT_INST_REG_ADDR(0),
+	.irq_no = DT_INST_IRQN(0),
+	.alts_list = peci_alt_inst,
+};
+
+static struct peci_it8xxx2_data peci_it8xxx2_data0;
+
+/* ITE IT8XXX2 PECI Functions */
+
+static void peci_it8xxx2_set_dvie(const struct device *dev, bool en)
+{
+	const struct peci_it8xxx2_config *config = dev->config;
+	struct peci_it8xxx2_regs *const peci_regs =
+		(struct peci_it8xxx2_regs *)config->base_addr;
+
+	if (en) {
+		peci_regs->PADCTLR |= PECI_DVIE;
+	} else {
+		peci_regs->PADCTLR &= ~PECI_DVIE;
+	}
+}
+
+static void peci_it8xxx2_init_vtts(const struct device *dev,
+					enum peci_vtts vol_opt)
+{
+	const struct peci_it8xxx2_config *config = dev->config;
+	struct peci_it8xxx2_regs *const peci_regs =
+		(struct peci_it8xxx2_regs *)config->base_addr;
+
+	peci_regs->PADCTLR = (peci_regs->PADCTLR & PECI_DVIE) | vol_opt;
+}
+
+static void peci_it8xxx2_rst_status(const struct device *dev)
+{
+	const struct peci_it8xxx2_config *config = dev->config;
+	struct peci_it8xxx2_regs *const peci_regs =
+		(struct peci_it8xxx2_regs *)config->base_addr;
+
+	peci_regs->HOSTAR = HOSTAR_RST_ANYBIT;
+}
+
+static int peci_it8xxx2_check_host_busy(const struct device *dev)
+{
+	const struct peci_it8xxx2_config *config = dev->config;
+	struct peci_it8xxx2_regs *const peci_regs =
+		(struct peci_it8xxx2_regs *)config->base_addr;
+
+		return (peci_regs->HOSTAR & HOBY) ? (-EBUSY) : 0;
+
+}
+
+static int peci_it8xxx2_check_host_finish(const struct device *dev)
+{
+	struct peci_it8xxx2_data *data = dev->data;
+	const struct peci_it8xxx2_config *config = dev->config;
+	struct peci_it8xxx2_regs *const peci_regs =
+		(struct peci_it8xxx2_regs *)config->base_addr;
+
+	int ret = k_sem_take(&data->device_sync_sem, K_MSEC(PECI_TIMEOUT_MS));
+
+	if (ret == -EAGAIN) {
+		LOG_ERR("%s: Timeout", __func__);
+		return -ETIMEDOUT;
+	}
+
+	if (peci_regs->HOSTAR != FINISH) {
+		printk("[PECI] Error: HOSTAR=0x%02X\r\n", peci_regs->HOSTAR);
+			return -EIO;
+	}
+
+	return 0;
+}
+
+static int peci_it8xxx2_configure(const struct device *dev, uint32_t bitrate)
+{
+	struct peci_it8xxx2_data *data = dev->data;
+	const struct peci_it8xxx2_config *config = dev->config;
+	struct peci_it8xxx2_regs *const peci_regs =
+		(struct peci_it8xxx2_regs *)config->base_addr;
+
+	uint8_t hoctl2r_to_write;
+
+	data->bitrate =  bitrate;
+
+	hoctl2r_to_write =
+		(peci_regs->HOCTL2R) & (~(PECI_IT8XXX2_BITRATE_BITS_MASK));
+
+	switch (bitrate) {
+	case PECI_IT8XXX2_BITRATE_2MHZ:
+		break;
+
+	case PECI_IT8XXX2_BITRATE_1MHZ:
+		hoctl2r_to_write |= PECI_IT8XXX2_BITRATE_1MHZ_BITS;
+		break;
+
+	case PECI_IT8XXX2_BITRATE_1P6MHZ:
+		hoctl2r_to_write |= PECI_IT8XXX2_BITRATE_1P6MHZ_BITS;
+		break;
+
+	default:
+		printk("[PECI] Error: Specified Bitrate Not Supported\r\n");
```

Comment:
Use logging here instead.  Also return -ENOTSUP.

Reply:
Got it, sir.
I'll change it to logging from now on.

Truly grateful.